### PR TITLE
Add config paths for microk8s

### PR DESCRIPTION
**Pull request by mkretz**
For running my own experiments I have added config paths for [Canonical's MicroK8s](https://github.com/ubuntu/microk8s). What do you think? Would it make sense to merge these to upstream?

### CODE REVIEW

File: cfg/config.yaml [hunk 1]
```diff
@@ -131,14 +140,17 @@ node:
       - "hyperkube kube-proxy"
       - "proxy"
     confs:
+      - /var/snap/microk8s/current/args/kube-proxy
```
Comment (lizrice) [inline cfg/config.yaml]: Please let's put this one as the last option in the list (they get checked in order)

**Comment by lizrice** [comment]
hi @mkretz are you still interested in finishing this one off?

**Comment by mkretz** [comment]
Hi @lizrice, sorry for the late reply. 

Absolutely! However, I think we should have someone from the [microk8s](https://github.com/ubuntu/microk8s) team review these paths to check whether any of them are likely to change in future releases. I have [opened an issue in that project](https://github.com/ubuntu/microk8s/issues/1001), asking whether one of the maintainers would have time for a short review. Ok with you?

**Comment by ktsakalozos** [comment]
Thank you for doing this @mkretz. It gives us the motivation to harden MicroK8s. 

The paths are correct and they are not going to change in the near future.

**Comment by lizrice** [comment]
Great so @mkretz if you wouldn't mind just moving the reference to `/var/snap/microk8s/current/args/kube-proxy` to be the last (rather than the first) in its list, we can merge this (per my review comment). Thanks!

**Comment by mkretz** [comment]
Done, @lizrice ! I have also fixed the checks (yaml linting issue). Let me know if anything else needs to be done. Thanks for adopting these!

**Pull request merged**